src/haddock/modules/analysis/caprieval/defaults.yaml

```diff
@@ -62,6 +62,19 @@ dockq:
   group: analysis
   explevel: easy
 
+global_rmsd:
+  default: true
+  type: boolean
+  title: Calculate global RMSD
+  short: Calculate full structure RMSD.
```

Comment:
Does this mean all heavy atoms RMSD?

Reply:
No, this means backbone RMSD (with default parameters), with the exception of small-molecules
For all heavy atoms RMSD, one should turn on the `allatoms = true` parameter.
Maybe I should update the long description to emphasize this behavior